config/environments/staging.rb

```diff
@@ -2,5 +2,7 @@
 if ENV["DISABLE_SSL"]
   Tahi::Application.configure do
     config.force_ssl = false
+
+    config.basic_auth_required = true
```

Comment:
Hm, we should bring `if ENV["DISABLE_SSL]` inside the `Tahi::Application.configure do` block. How about this:

```ruby
Tahi::Application.configure do
  config.force_ssl = false if ENV["DISABLE_SSL"]

  config.basic_auth_required = true
end